addons/hr_expense/models/hr_expense.py

```diff
@@ -919,12 +919,11 @@ def _compute_amount(self):
         for sheet in self:
             sheet.total_amount = sum(sheet.expense_line_ids.mapped('total_amount_company'))
 
-    @api.depends("account_move_id.line_ids")
+    @api.depends('account_move_id.amount_residual_signed')
```

Comment:
is there a reason the depends here is on the `amount_residual_signed` and the compute on the hr.expense is on the `amount_residual` ? (I guess it is equivalent)

Reply:
also: use either single or double quotes, but choose one for this PR :)

Reply:
oops :p

Reply:
you did not answered my first question :D

Reply:
I answered in the other message
Because signed is in company currency, (move) amount_residual isn't
Expense sheet amount residual is always in company currency
The expense isn't always in company currency